vocode/streaming/audio/factory.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from typing import Optional
+from vocode.streaming.models.audio import AudioServiceConfig
+from vocode.streaming.audio.audio_service import AudioService
+
+
+class AudioServiceFactory:
+    def create_audio_service(
+        self,
+        conversation_id: str,
+        audio_service_config: AudioServiceConfig,
+        logger: Optional[logging.Logger] = None,
+    ):
+        if isinstance(audio_service_config, AudioServiceConfig):
```

Comment:
given `audio_service_config: AudioServiceConfig` in the function definition, do we need to check that this is an instance of it each time?

Reply:
right now, no need (this is for future we have separate audio_services, we need to check config and create related class). Added as place holder for future.